### PR TITLE
add in multiple file format support to rationale

### PR DESCRIPTION
**Pull request by kestewart**
Update Rationale section to specify the new file formats,  clarify the use cases being supported by SPDX, and update the changes since 2.1 specification.

This satisfies https://github.com/spdx/spdx-spec/issues/200

### CODE REVIEW

File: chapters/1-rationale.md [hunk 1]
```diff
@@ -70,11 +70,17 @@ In an SPDX document, Relationship elements can be used to indicate relationships
...
 **1.7.5** Must permit automated specification syntax validation.
 
-**1.7.6** Resource Description Framework (RDF) can be used to represent this information, as can an annotated tag value flat text file.
+**1.7.6** Multiple file formats can be used to represent the information being exchanged.   Current supported formats include:
+* YAML 1.2  see: https://yaml.org/spec/1.2/spec.html
+* JavaScript Object Notation (JSON) see: ECMA-404(https://www.ecma-international.org/publications/files/ECMA-ST/ECMA-404.pdf)
+* Extensible Markup Language (XML) see: https://www.w3.org/TR/2008/REC-xml-20081126/
+* Resource Description Framework (RDF also referred to as RDF/XML) see: https://www.w3.org/TR/rdf-syntax-grammar/
+* `tag:value` flat text file as described in this specification
+* .xls spreadsheets
```
Comment (swinslow) [inline chapters/1-rationale.md]: I would suggest removing .xls spreadsheets from the list. Even if the Java tools might handle them, I don't believe they're defined in the spec or included in the new `examples/` folder like the new JSON / YAML / XML formats.

File: chapters/1-rationale.md [hunk 2]
```diff
@@ -90,7 +96,7 @@ In an SPDX document, Relationship elements can be used to indicate relationships
 
 ## 1.8 Conformance <a name="1.8"></a>
 
-**1.8.1** A file can be designated an SPDX document, if it is compliant with the requirements of the SPDX Trademark License (See the SPDX Trademark Page).
+**1.8.1** A file can be designated an SPDX document, if it is compliant with the requirements of the SPDX Trademark License (See the SPDX Trademark Page on the spdx.org web site).
 
 **1.8.2** The official copyright notice to be used with any verbatim reproduction and/or distribution of this SPDX Specification 2.2-DRAFT is:
```
Comment (swinslow) [inline chapters/1-rationale.md]: Should "-DRAFT" be taken out now? Or are we separately tracking this elsewhere to make sure that it's removed before the new version is released?

File: chapters/1-rationale.md (hunk 1 above)
Comment (kestewart) [reply chapters/1-rationale.md]: Spreadsheet format on WIKI page will need to be updated (move to github repo,  first 2.1 version,  then apply 2.2 after OWL).

Comment (kestewart) [reply chapters/1-rationale.md]: Key is to make sure we have an examples present.

**Pull request merged**